ShowMeTheXAML/Uno.ShowMeTheXAML.csproj

```diff
@@ -19,6 +19,10 @@
     <DefineConstants>$(DefineConstants);__WASM__</DefineConstants>
   </PropertyGroup>
 
+  <ItemGroup>
```

Comment:
Why this reference?

Reply:
If it's not too much code to bring in, let's try and not depend on it :)

Reply:
solely for `Disposable.Create(Action)` used in here: https://github.com/unoplatform/ShowMeTheXAML/pull/4/files#diff-5f903cd6c33fb5c0cf2f0332e190903bR202 
i guess we could let the nice syntax go.